--- lib/lib/Makefile.am.link	2011-04-14 10:14:50.000000000 +0200
+++ lib/lib/Makefile.am	2011-04-14 10:16:03.000000000 +0200
@@ -6,6 +6,7 @@
 xneur_include_HEADERS = xneur.h		
 
 libxneur_la_SOURCES =		\
+	../config/xnconfig_files.c \
 	xneurlib.c		
 	xneur.h		
 
--- lib/config/Makefile.am.link	2011-04-14 10:16:53.000000000 +0200
+++ lib/config/Makefile.am	2011-04-14 10:17:15.000000000 +0200
@@ -10,7 +10,6 @@
 	xnconfig_memory.h	
 
 libxnconfig_la_SOURCES =	\
-	xnconfig_files.c	\
 	xnconfig.c		\
 	xnconfig_memory.c	
 
--- plugins/statistic/Makefile.am.link	2011-04-14 10:20:43.000000000 +0200
+++ plugins/statistic/Makefile.am	2011-04-14 10:21:36.000000000 +0200
@@ -8,6 +8,5 @@
 	statistic.c
 
 libxnstatistic_la_CFLAGS = -I@top_srcdir@/lib/config -I@top_srcdir@/lib/lib @DEFAULT_CFLAGS@
-libxnstatistic_la_LDFLAGS = @X11_LIBS@
 	 
-libxnstatistic_la_LIBADD = 
+libxnstatistic_la_LIBADD = @X11_LIBS@ ../../lib/config/libxnconfig.la
